src/components/Photo/DocumentLiveCapture.js

```diff
@@ -0,0 +1,125 @@
+// @flow
+import * as React from 'react'
+import { h, Component } from 'preact'
+import { takePhoto } from '~utils/camera.js'
+import { mimeType } from '~utils/blob.js'
+import { DocumentOverlay } from '../Overlay'
+import { ToggleFullScreen } from '../FullScreen'
+import Spinner from '../Spinner'
+import Timeout from '../Timeout'
+import Camera from '../Camera'
+import CameraError from '../CameraError'
+import style from './style.css'
+
+type State = {
+  hasBecomeInactive: boolean,
+  hasCameraError: boolean,
+  isLoading: boolean
+}
+
+type Props = {
+  translate: (string, ?{}) => string,
+  onCapture: Function,
+  renderFallback: Function,
+  isUploadFallbackDisabled: boolean,
+  trackScreen: Function,
+  documentType: string,
+  className: string,
+  containerClassName: string,
+  renderTitle: Function,
+  renderError: Function
+}
+
+export default class DocumentLiveCapture extends Component<Props, State> {
+  webcam = null
+
+  state: State = {
+    hasBecomeInactive: false,
+    hasCameraError: false,
+    isLoading: false
+  }
+
+  handleTimeout = () => this.setState({ hasBecomeInactive: true })
```

Comment:
There is a lot of code shared here shared between this component and the selfie photo one. For example, this camera errors are very similar and the capture part as well. Maybe we can extract it into a component

Reply:
Yeah, that's true. I guess it can be done as a separate ticket.